modules/Vtiger/ExecuteFunctions.php

```diff
@@ -693,6 +693,31 @@
 			);
 		}
 		break;
+	case 'getDeepFieldValue':
+		$moduleName = vtlib_purify($_REQUEST['moduleName']);
+		$value = vtlib_purify($_REQUEST['value']);
+		$fieldsArray = explode("-&gt;", $value);
+		// remove all $ signs
+		foreach ($fieldsArray as &$value) {
```

Comment:
if substitute this loop with array_walk, would it be faster?

`array_walk($fieldsArray, function (&$item, $key) { $item = substr($item, 1); });`